README.md

```diff
@@ -93,6 +93,34 @@ Building clang from sources takes time and resources and it can be avoided:
   moment, see [KhronosGroup/SPIRV-LLVM-Translator#477](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/477)
   to track progress, discuss and contribute.
 
+### Build with SPIRV-Tools
+
+The translator can use SPIRV-Tools to generate assembly with conformant syntax.
```

Comment:
It would be nice to have a link to the SPIR-V Tools repo.
```suggestion
The translator can use [SPIRV-Tools](https://github.com/KhronosGroup/SPIRV-Tools) to generate assembly with conformant syntax.
```